src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -350,8 +266,8 @@ public String toJSON(MaxwellOutputConfig outputConfig) throws Exception {
 		}
 
 		DataJsonGenerator dataWriter = outputConfig.encryptionMode == EncryptionMode.ENCRYPT_DATA
-			? encryptingJsonGeneratorThreadLocal.get()
-			: plaintextDataGeneratorThreadLocal.get();
+			? json.getEncryptingGenerator()
+			: json.getPlaintextGenerator();
```

Comment:
i see.  well, in the future this would be better if `MaxwellJson` just returned some kind of generator that had a unified interface, but this is good enough for now.

Reply:
I considered subclassing JsonGenerator, but it's a pretty big interface, so keeping them separate was less work and makes it clear what functionality MaxwellJson actually adds. I also kind of like that you can only access the JsonGenerator by calling `reset()`, which
discourages fly-by writing to an effectively global buffer. But maybe that's me justifying my laziness ;)